UI/forms/OBSBasic.ui

```diff
@@ -168,15 +168,15 @@
              <string>StudioMode.PreviewSceneName</string>
             </property>
             <property name="alignment">
-             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
+             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
             <property name="themeID" stdset="0">
              <string>previewProgramLabels</string>
             </property>
            </widget>
           </item>
           <item>
-           <widget class="OBSBasicPreview" name="preview">
+           <widget class="OBSBasicPreview" name="preview" native="true">
```

Comment:
Need someone on Mac to test if this causes issues. I remember a long time ago there was some horrific issue on Mac caused by Qt adding this attribute that was mitigated by removing it.

Reply:
You might be thinking of Linux? I didn't find anything at a glance for macOS.

https://github.com/obsproject/obs-studio/commit/9ee7547e192fac25cf08124d57037cb124b69c38

Reply:
cc @PatTheMav @gxalpha I don't recall if we've ever had issues with native widgets on macOS?
cc @GeorgesStavracas See above.

#3782 did previously remove `native="true"` from this widget.
https://github.com/obsproject/obs-studio/commit/9ee7547e192fac25cf08124d57037cb124b69c38#diff-c4723e7657a9d380fa8407cf136dc7e888e73120f82b0552722d45a709f7120fL170-R170

Reply:
These native surfaces were causing trouble with Wayland back in the day. Not sure how that goes now.

Reply:
Are we feeling adventurous to re-examine this? The way I see it, we have a few paths forward:
1. Test this on Wayland before merging.
2. Revert just these changes for now. Test the `native="true"` changes on Wayland later. Open a new PR for those later if testing goes well.
3. Merge it and be on the lookout for issues. Revert the `native="true"` changes if needed.

I would like this to not get blocked for a prolonged period of time.

Reply:
My personal preference is (2) but it's more laborious than the others. If it's too much trouble, just merge with it.

Reply:
Hmm. Thanks. Option 2 is probably "safest", but does also require _someone_ to go test on Wayland and check for issues and then PR/merge those changes at a later date, which is currently not in my capacity to do so. @derrod ?

Reply:
Looks fine on GNOME Wayland after a quick test resizing arround with Studio mode on and off.

Reply:
Sounds like we'll give this a scream test then.